orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/states/SaltStates.java

```diff
@@ -407,4 +407,10 @@ private static String parseVersion(String versionCommandOutput, Optional<String>
         }
         return versionCommandOutput;
     }
+
+    public static boolean unboundClusterConfigPresentOnAnyNodes(SaltConnector sc, Target<String> target) {
+        return measure(() -> sc.run(target, "file.file_exists", LOCAL, PingResponse.class, "/etc/unbound/conf.d/00-cluster.conf"), LOGGER,
```

Comment:
PingResponse.class? Shouldn't it be FileExistsSaltResponse?

Reply:
I just saw you created FileExistsSaltResponse and it is not used if I am correct, I thought you forgot to change it.

Reply:
Ok I can see it is a test related file

Reply:
Then it is fine for me